data/humans/obsolete-occupations.json

```
@@ -0,0 +1,173 @@
+{
+	"description":"A list of obsolete occupations, adapted from https://en.wikipedia.org/wiki/Category:Obsolete_occupations",
+	"occupations": [
+		"alchemist",
+		"alewife",
+		"amtmann",
+		"arendator",
+		"arrephoros",
+		"ash burner",
+		"baby farmer",
+		"badger",
+		"beamer",
+		"bear-leader",
+		"bematist",
+		"between maid",
+		"bobbin boy",
+		"bombardier",
+		"book peddler",
+		"breaker boy",
+		"broomsquire",
+		"bullocky",
+		"burlak",
+		"cabin boy",
+		"castrato",
+		"cavalry",
+		"chapman",
+		"charcoal burner",
+		"chopchurch",
+		"cigarette girl",
+		"city guard",
+		"coachman",
+		"colportage",
+		"cooper",
+		"cornjerker",
+		"court dwarf",
+		"crossing sweeper",
+		"cuirassier",
+		"cup-bearer",
+		"dahomey amazon",
+		"deal porter",
+		"doffer",
+		"dog whipper",
+		"donkey puncher",
+		"dragoon",
+		"drummer",
+		"drysalter",
+		"dyachok",
+		"dyak",
+		"elevator operator",
+		"engagé",
+		"exilarch",
+		"expressman",
+		"fangshi",
+		"feeder of lice",
+		"fendersmith",
+		"fifer",
+		"footman",
+		"francs-tireur",
+		"freischar",
+		"galley slave",
+		"gandy dancer",
+		"glimmer man",
+		"gong farmer",
+		"groom of the Stool",
+		"grubber",
+		"gungnyeo",
+		"hall boy",
+		"harquebusier",
+		"heavy infantryman",
+		"herb strewer",
+		"hessian",
+		"hetaira",
+		"hippei",
+		"hiwi",
+		"hoplite",
+		"computer",
+		"hurrier",
+		"ice cutter",
+		"iceman",
+		"immortals",
+		"impus",
+		"instigator regnus",
+		"investigating censor",
+		"kenin",
+		"khapper",
+		"knocker-up",
+		"lady's companion",
+		"lamp trimmer",
+		"lamplighter",
+		"law-writer",
+		"leech collector",
+		"limner",
+		"line infantryman",
+		"link-boy",
+		"Łowczy",
+		"marchand-mercier",
+		"master of the Buckhounds",
+		"matagi",
+		"mounted infantryman",
+		"mud clerk",
+		"mudlark",
+		"mule scavenger",
+		"münzmeister",
+		"mursmäcka",
+		"ninja",
+		"nursemaid",
+		"packetman",
+		"paedagogus",
+		"peddler",
+		"pevchy dyak",
+		"pinsetter",
+		"plague doctor",
+		"podyachy",
+		"portreeve",
+		"postilion",
+		"poundmaster",
+		"powder monkey",
+		"priest hunter",
+		"printer's devil",
+		"privateer",
+		"professional mourner",
+		"punkah wallah",
+		"ragpicker",
+		"reeve",
+		"resident",
+		"resin worker",
+		"retinue",
+		"rhapsode",
+		"rower woman",
+		"runemaster",
+		"rynda",
+		"samurai",
+		"sandal-bearer",
+		"sanggung",
+		"scribe",
+		"scrivener",
+		"scullery maid",
+		"scutelnic",
+		"seeress",
+		"seneschal",
+		"situ",
+		"skomorokh",
+		"slave catcher",
+		"soda jerk",
+		"sōhei",
+		"spearman",
+		"stavilac",
+		"still room maid",
+		"sugar-baker",
+		"swabian child",
+		"switchboard operator",
+		"taffeltäckare",
+		"tawaif",
+		"tea lady",
+		"teamster",
+		"telegraphist",
+		"thief-taker",
+		"toad doctors",
```

Let's put this as singular to match the others? 
```suggestion
		"toad doctor",
```
(What on earth is a toad doctor?!)